pom.xml

```diff
@@ -273,256 +319,281 @@
-	
+      <groupId>org.apache.chemistry.opencmis</groupId>
+      <artifactId>chemistry-opencmis-client-impl</artifactId>
+      <version>0.6.0</version>
+    </dependency>
+
     <dependency>
-        <groupId>javax.mail</groupId>
-	<artifactId>mail</artifactId>
-	<version>1.4</version>
+      <groupId>eu.agno3.jcifs</groupId>
+      <artifactId>jcifs-ng</artifactId>
+      <version>2.0.5</version>
     </dependency>
-	
-	
+
     <dependency>
-        <groupId>com.googlecode.sardine</groupId>
-        <artifactId>sardine</artifactId>
-        <version>343</version>
+      <groupId>javax.mail</groupId>
+      <artifactId>mail</artifactId>
+      <version>1.4</version>
     </dependency>
-    
+
+
     <dependency>
-  		<groupId>org.apache.httpcomponents</groupId>
-  		<artifactId>httpclient</artifactId>
-  		<version>4.1.2</version>
-	</dependency>
-	
-	<dependency>
-  		<groupId>org.slf4j</groupId>
- 		<artifactId>slf4j-api</artifactId>
-  	    <version>1.6.2</version>
-	</dependency>
+      <groupId>com.googlecode.sardine</groupId>
+      <artifactId>sardine</artifactId>
+      <version>343</version>
+    </dependency>
 
     <dependency>
-    	<groupId>org.aspectj</groupId>
-    	<artifactId>aspectjrt</artifactId>
-    	<version>1.7.3</version>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpclient</artifactId>
+      <version>4.1.2</version>
     </dependency>
 
     <dependency>
-        <groupId>org.aspectj</groupId>
-        <artifactId>aspectjweaver</artifactId>
-        <version>1.7.3</version>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <version>1.6.2</version>
     </dependency>
 
     <dependency>
-    	<groupId>org.springframework</groupId>
-    	<artifactId>spring-webmvc-portlet</artifactId>
-    	<version>3.1.1.RELEASE</version>
+      <groupId>org.aspectj</groupId>
+      <artifactId>aspectjrt</artifactId>
+      <version>1.7.3</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.aspectj</groupId>
+      <artifactId>aspectjweaver</artifactId>
+      <version>1.7.3</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-webmvc-portlet</artifactId>
+      <version>3.1.1.RELEASE</version>
+    </dependency>
+
+    <dependency>
+      <groupId>javax.servlet</groupId>
+      <artifactId>jstl</artifactId>
+      <version>1.1.2</version>
     </dependency>
- 
-     <dependency>
-    	<groupId>javax.servlet</groupId>
-    	<artifactId>jstl</artifactId>
-    	<version>1.1.2</version>
+
+    <dependency>
+      <groupId>taglibs</groupId>
+      <artifactId>standard</artifactId>
+      <version>1.1.2</version>
     </dependency>
-    
+
     <dependency>
-    	<groupId>taglibs</groupId>
-        <artifactId>standard</artifactId>
-		<version>1.1.2</version>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-vfs2</artifactId>
+      <version>2.0-esup-patch-2</version>
```

Comment:
Need any help upstreaming the patches so a forked version isn't needed?

Reply:
@ChristianMurphy : Yes it would be great but the problem is on commons-vfs lib, there isn't more an active work on.
For more details on the needs and talks see the other PR #41

Reply:
Okay, so it looks like the only blocker to going back to the official version is getting https://github.com/apache/commons-vfs/pull/29 merged?

Reply:
> there isn't more an active work on.

Hmm, looks like the last change merged 23 hours ago, seems somewhat active.
https://github.com/apache/commons-vfs/commits/trunk

Reply:
Just a heads up, review has started on 
https://github.com/apache/commons-vfs/pull/29